Backstage icon fix

jdlrobson opened the pull request:
Fixes css issues introduced by pmario's changes (included)
This extends the work by pmario on his branch with the same name.

pmario committed:
modified:   src/icons/backstage.svg.tid
modified:   src/styles/StyleSheetBackstage.tid
backstage.svg causes a clickable area, that is way bigger than the icon. fixed it and fixed CSS as well.

jdlrobson committed:
define size of open backstage icon in css

StyleSheetBackstage now defines the size of the backstage icon, not
the TiddlySpaceBackstage plugin. This fixes the problems introduced by
pmarios svg refactorings.

pmario commented:
Hi Jon,
Is there still a margin-top: -3 px needed to look nice?
May be there is still something wrong with the svg?

jdlrobson committed:
Merge branch 'master' of git://github.com/TiddlySpace/tiddlyspace into backstageIconFix

jdlrobson commented:
I don't believe so. It is rendering correctly for me with these changes - I believe your svg cleanup has removed the need for that margin which is great.

Unless you know something that I do not... feel free to upload a photo on:
http://pastebin.tiddlyspace.com/

if this is the case.

pmario commented on src/styles/StyleSheetBackstage.tid:
```diff
@@ -77,9 +77,10 @@ modifier: osmosoft
 
-#backstageShow {
-	margin-right: -3px;
-	margin-top: -3px;
```
uups. sry I didn't realize the - in front

pmario commented on src/styles/StyleSheetBackstage.tid:
```diff
@@ -77,9 +77,10 @@ modifier: osmosoft
+	height: 24px;
 }
 
 #backstageShow .svgIcon {
```
I thought, this part will go away also?
Is there any need for CSS tweaking the svg here?
Is it the close button, if the backstage is open?

jdlrobson replied:
The size is now defined here in the css rather than in the plugin. If no dimensions are given it will default to height 100% on which case it will look odd on some browsers. Sounds like this is thus good to go.

jdlrobson replied:
Apologies I misunderstood which line you were referring to. You are correct, it is unnecessary. I've updated the pull request accordingly. Good eyes! :)

jdlrobson committed:
removed redundant styling rules

These rules have no effect and are not needed.
Thanks to pmario for pointing out.
